cherrytrack/helpers/plates.py

```diff
@@ -82,13 +87,22 @@ def get_wells_for_destination_plate(destination_barcode: str) -> List[Dict[str,
             {
                 "automation_system_run_id": row.automation_system_run_id,
                 "destination_coordinate": row.destination_coordinate,
+                "date_picked": row.date_picked or "",
+                "created_at": row.created_at or "",
                 **get_well_content(row),
             }
         )
 
     return samples
 
 
+def get_well_picked_status(row):
+    if row.automation_system_run_id:
```

Comment:
Don't think this logic is correct.
If this is the destination plate, the run id will be set for every destination well row when the plate is first scanned on the robot. The wells are empty at that point (not picked into yet).
A destination well is picked into if either the source_plate_well_id or the control_plate_well_id are not null. It is not picked into yet if both are null.
The run id is updated when the well is picked into, in case the lab users partially fill the plate in one run, then fill in the rest in another.